docs/source/conf.py

```diff
@@ -53,13 +51,10 @@
 # If true, `todo` and `todoList` produce output, else they produce nothing.
 todo_include_todos = True
 
-# Setup Sphinx
-def setup(sphinx):
-    sphinx.add_config_value('recommonmark_config', {
-        'enable_eval_rst': True,
-        'enable_auto_toc_tree': False,
-    }, True)
-    sphinx.add_transform(AutoStructify)
+# -- Options for myst parser ----------------------------------------
+
+myst_enable_extensions = ["colon_fence"]
+suppress_warnings = ["myst.header", "ref.myst"]
```

Comment:
not sure we should just suppress those, at least without a tracking issue

Reply:
https://github.com/scylladb/scylla-operator/issues/1421
https://github.com/scylladb/scylla-operator/issues/1422
done